src/WhatsApp.php

```diff
@@ -121,4 +121,28 @@ public function createBody($to, $templateName, $languageCode, $components = [],
 
         return $body;
     }
+
+     /**
+     * @param string $whatsAppBusinessAccountId
+     * 
+     * @param string $accessToken
+     *
+     * @param array $filters
+     *   
+     * @return array|mixed
+     */
+    public function getMessageTemplates($whatsAppBusinessAccountId, $accessToken, $filters = []){
+        $queryAppends = '';
+        if(!empty($filters)){
+            $queryAppends = $queryAppends. '&'.http_build_query($filters);
+        }
+        $response = Http::get(config('whatsApp.api_uri') . $whatsAppBusinessAccountId . '/message_templates?access_token=' . $accessToken.$queryAppends);
```

Comment:
IMHO, we should remove `&` from the above `$queryAppends` and add it here between `$queryAppends` and `$accessToken`.

Reply:
@binal-7span 
Here we need to append and `&` before `http_build_query($filters)`. 
If we append `&` as per your suggestion, the URL will be as below...
```
/message_templates?access_token=<access_token>&
```